bvm/ballerina-core/src/main/java/org/ballerinalang/bre/bvm/CPU.java

```diff
@@ -821,6 +826,14 @@ private static void tryExec(WorkerExecutionContext ctx) {
         }
     }
 
+    private static void createClone(int[] operands, WorkerData sf) {
+        int i = operands[0];
+        int j = operands[1];
+        if (isAnydata(sf.refRegs[i].getType())) {
```

Comment:
can this be false? if it can be false, shouldn't we print throw a runtime exception for else case?